app/models/profile.rb

```diff
@@ -9,6 +9,11 @@ class Profile < ApplicationRecord
 
   store_attribute :data, :custom_attributes, :json, default: {}
 
+  # Static fields are columns on the profiles table; they have no relationship
+  # to a ProfileField record. These are columns we can safely assume exist for
+  # any profile on a given Forem.
+  STATIC_FIELDS = %w[summary location website_url].freeze
```

Comment:
I often prefer calling methods over directly accessing constants from other classes:

```ruby
def self.static_fields
  STATIC_fields
end
```

This way we can change it later to e.g. something like 

```ruby
FILTERED_COLUMNS =  %w[id user_id data created_at updated_at].freeze

def self.static_fields
  Profile.column_names - FILTERED_COLUMNS
end
```

Reply:
Thanks for making this change @jacobherrington! Another advantage forgot to mention is that stubbing methods in tests feels much "cleaner" than stubbing constants.